src/main/java/org/folio/fqm/migration/strategies/V0POCMigration.java

```diff
@@ -93,7 +93,6 @@ public class V0POCMigration extends AbstractSimpleMigrationStrategy {
     Map.entry("instance_statistical_code_ids", "instance.statistical_code_ids"),
     Map.entry("instance_statistical_codes", "instance.statistical_code_names"),
     Map.entry("instance_status_id", "instance.status_id"),
-    Map.entry("instance_status", "instance.status"),
```

Comment:
This should be something like `inst_stat.name`, shouldn't it?

Reply:
We still need to migrate the field name, just need to change it to the new one since `status` is not part of `instance` ET anymore